pd-recover.md

```diff
@@ -6,42 +6,152 @@ category: reference
 
 # PD Recover User Guide
 
-PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally.
+PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally. PD Recover is downloaded with tidb-ansible in `resource/bin/pd-recover`.
 
-## Source code compiling
+## Quick Start
 
-1. [Go](https://golang.org/) Version 1.9 or later
-2. In the root directory of the [PD project](https://github.com/pingcap/pd), use the `make` command to compile and generate `bin/pd-recover`
+This section describes how to use PD Recover to recover a PD cluster.
 
-## Usage
+### Get cluster ID
 
-This section describes how to recover a PD cluster which cannot start or provide services normally.
+The cluster ID can be obtained in the log of PD, TiKV or TiDB. You can either use `ansible ad-hoc` in the Control Machine, or view the log directly on the server.
 
-### Flags description
+#### Get [info] cluster ID from PD log (recommended)
 
+To get the `[info]` cluster ID from the PD log, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+```
+ansible -i inventory.ini pd_servers -m shell -a 'cat {{deploy_dir}}/log/pd.log | grep "init cluster id" | head -10'
+```
+
+```
+10.0.1.13 | CHANGED | rc=0 >>
+[2019/10/14 10:35:38.880 +00:00] [INFO] [server.go:212] ["init cluster id"] [cluster-id=6747551640615446306]
+……
+```
+
+#### Get [info] cluster ID from TiDB log
+
+To get the `[info]` cluster ID from the TiDB log, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+```
+ansible -i inventory.ini tidb_servers -m shell -a 'cat {{deploy_dir}}/log/tidb*.log | grep "init cluster id" | head -10'
+```
+
+```
+10.0.1.15 | CHANGED | rc=0 >>
+2019/10/14 19:23:04.688 client.go:161: [info] [pd] init cluster id 6747551640615446306
+……
+```
+
+#### Get [info] PD cluster from TiKV log
+
+To get the `[info]` PD cluster from the TiKV log, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+```
+ansible -i inventory.ini tikv_servers -m shell -a 'cat {{deploy_dir}}/log/tikv* | grep "PD cluster" | head -10'
+```
+
+```
+10.0.1.15 | CHANGED | rc=0 >>
+[2019/10/14 07:06:35.278 +00:00] [INFO] [tikv-server.rs:464] ["connect to PD cluster 6747551640615446306"]
+……
+```
+
+### Get Alloc ID (TiKV StoreID)
+
+When you specify `alloc-id`, you need to specify a value greater than the currently largest `Alloc ID`. You can either use `ansible ad-hoc` in the Control Machine, or view the log directly on the server.
```

Comment:
@shafreeck Please check whether my comment is technically correct, thanks!
```suggestion
The `alloc-id` value you specify must be larger than the currently largest `Alloc ID` value. To get `Alloc ID`, you can either use the `ansible ad-hoc` command in the Control Machine, or view the log directly on the server.
```

Reply:
LGTM